system/cfme-setup.sh

```diff
@@ -31,6 +31,17 @@ cat <<'EOF' > /etc/httpd/conf.d/ssl.conf
 # upgraded.
 EOF
 
+# Incorporate the iptable updates
+if [[ -f /etc/sysconfig/iptables.miq ]]; then
+  /sbin/iptables-save > /etc/sysconfig/iptables.SYSTEM
+  /sbin/iptables-restore /etc/sysconfig/iptables.miq
+  /sbin/service iptables save
+  # Expect no more than the four datestamp differences.
+  if [[ $(/usr/bin/diff -Ew -U 0 /etc/sysconfig/iptables /etc/sysconfig/iptables.miq  | /bin/grep "^@@" | /usr/bin/wc -l) -gt 4 ]]; then
```

Comment:
I don't understand the need for this conditional at all, and the comment `# Expect no more than the four datestamp differences.` doesn't make it any clearer.  What does four datestamps signify...i.e. why 4?